linker_scripts/object_script.txt

```diff
@@ -32,3 +32,7 @@ D_0601F050 = 0x0601F050;
 D_06018DA0 = 0x06018DA0;
 D_0601EF10 = 0x0601EF10;
 D_06018C60 = 0x06018C60;
+
+/* arms_hook */
+D_0601D960 = 0x0601D960;
+D_040008D0 = 0x040008D0;
```

Comment:
Group this with the other `D_04XXXXXX` symbols. Since segment 4 is gameplay_keep, there could be other overlays that use this symbol. It would be better to group gameplay_keep's symbols together to avoid duplication.